src/tests/test_utils/common_test_utils/include/common_test_utils/data_utils.hpp

```diff
@@ -547,6 +547,52 @@ inline ov::float8_e5m2 ie_abs(const ov::float8_e5m2& val) {
     return ov::float8_e5m2::from_bits(val.to_bits() & 0x7F);
 }
 
+template <class T_ACTUAL, class T_EXPECTED>
```

Comment:
What about to use `/home/efode/repo/openvino/src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp::Compare` instead of adding new function?

Reply:
The main difference is Compare from ov_tensor_utils.cpp can operate only full tensors. And comparator I added can operate raw data of any length (e.g. compare a[1-5] with b[3-8]).

Of course we can unify approach and use one base function for both cases. But it requires a bit much efforts to implement and test.

Reply:
Let's try to use `ov_tensor_utils.cpp::compare()` - I hope it should be easy to create `ov::Tensor` over existed memory pointer with some element_type and size (aka shape)